examples/apps/cli/main.c

```diff
@@ -97,3 +98,22 @@ int main(int argc, char *argv[])
 
     return 0;
 }
+
+/*
+ * Provide, if required an "otPlatLog()" function
+ */
+#if ((OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_NONE) ||  (OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_APP_STREAM))
+void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat, ...)
```

Comment:
I did't miss NCP logs, i just didn't like to demand application to implement its own otPlatLog in case i want to run certification tests (which they have to be outputed in CLI/THCI interface).

Anyway i am ok with current approach, i will modify our platform log module to use otCliLogv function in case certification logs are enabled, else RTT. Then we will use OPENTHREAD_CONFIG_LOG_PLATFORM_DEFINED also if CERT_LOGS variable is set.

I will provide a pull request after your is merged.

Thanks!